### PR TITLE
Zonalwrapper

### PR DESCRIPTION
**Pull request by remeryAGS**

### CODE REVIEW

**Comment by demiurg** [comment]
Unless there is a version of models deployed of one of the intermediarry migrations, I recommend squashing them.

File: gips/inventory/orm/settings.py [hunk 1]
```diff
@@ -20,6 +20,9 @@
 ALLOWED_HOSTS = []
 
 
+# Needed for zonal_wrapper.py should probably move to local settings
+GIPSBIN_PATH = '/home/remery/Projects/GipsAlg/gips-bin/gipsbin'
```
Comment (ags-tolson) [inline gips/inventory/orm/settings.py]: This may not be portable.

Comment (remeryAGS) [inline gips/inventory/orm/settings.py]: Everyone should just structure their projects to look like this

**Comment by ags-tolson** [comment]
@demiurg you're right that it's a good idea to the migrations, but this branch is sort of a prototype so it's allowed to be messier than usual.

**Comment by ags-tolson** [comment]
So presently 18 unit tests break in this branch, but that's probably okay because of this branch's prototype status.

**Comment by ags-tolson** [comment]
the circular dep notwithstanding, this looks ok to merge to me

**Comment by ircwaves** [comment]
per @remeryAGS -- closed and will arise again.

**Pull request closed without merging**